site/en/docs/devtools/css/color/index.md

```diff
@@ -0,0 +1,157 @@
+
+   Or one defined by the [`color(<color_space> X X X)`](/articles/high-definition-css-color-guide/#the-color-function) function.
+
+### Convert colors {: #convert-colors }
+
+When you switch between color spaces with the **Display value switcher**, DevTools automatically converts the values.
+
+{% Aside 'caution' %}
+When converting from HD to non-HD, DevTools clips gamuts to fit the space and marks clipped values with {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hJxGejygnlBfMvhMtcCX.svg", alt="Warning.", width="24", height="24" %} warning icons. Hover over the icon to see the original value.
+
+{% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/gKaBwo9xmgtONghzOdZr.png", alt="A warning icon indicating a gamut clipping and a tooltip with the original value.", width="400", height="204", class="screenshot" %}
+{% endAside %}
+
+{% Video src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/wsWrvMTPYM0a5tmNdzlQ.mp4", controls="true", loop="true", muted="true", class="screenshot" %}
+
+## Fix contrast {: #fix-contrast }
+
+{% Aside 'important' %}
+The **Color Picker** calculates contrast ratios only for the `color` values and relative to the respective `background-color` values. So, only the `color` values have **Contrast ratio** sections.
+{% endAside %}
+
+To fix a contrast issue for a `color` declaration:
+
+1. Open the **Color Picker** next to the `color` value.
+1. Expand the **Contrast ratio** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/1UTR0tiSxiKD0YSwAc1g.svg", alt="Expand.", width="24", height="24" %} section.
+1. Use the suggested color that complies with a guideline:
+
+   - Click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/5sHRUwsqWoQHF2BiEwlm.png", alt="Use suggested color.", width="23", height="20" %} next to the guideline.
+   - In the **Shades** preview at the top, drag the **Color circle** below the corresponding line.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/1OBcwQNbZivVFL4vbC47.png", alt="The expanded contrast ratio section with WebAIM or APCA guidelines.", width="800", height="518" %}
+
+{% Aside %}
+By default, the **Color Picker** suggests you to follow the [WebAIM guidelines](https://webaim.org/standards/wcag/). Alternatively, you can set it to suggest values compliant with [APCA](https://web.dev/color-and-contrast-accessibility/#advanced-perceptual-contrast-algorithm-apca):
+
+In {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [**Settings** > **Experiments**](/docs/devtools/settings/experiments/), check {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hmp8j3HiLMCcqPArD9yt.svg", alt="Checkbox.", width="22", height="22" %} **Enable new Advanced Perceptual Contrast Algorithm (APCA)...**
+{% endAside %}
+
+To get a list of all contrast issues in one go, follow the [Make your website more readable](/docs/devtools/accessibility/contrast/) guide.
+
+## Sample a color anywhere with the Eyedropper {: #eyedropper }
```

Comment:
There's a new keyboard shortcut for sampling color. See the WNDT 111.

Reply:
Done